app/javascript/Components/student_table.jsx

```diff
@@ -95,8 +96,9 @@ class RawStudentTable extends React.Component {
           authenticity_token={this.props.authenticity_token}
         />
         <CheckboxTable
+          loading={this.state.loading}
           ref={r => (this.checkboxTable = r)}
-          data={data.students}
+          data={this.state.loading ? [] : data.students}
```

Comment:
This change is unnecessary (whether or not the loading spinner appears should depend only on the `loading` prop, and not the value of the `data` prop)